packages/core/tests/integration/components/navi-visualizations/table-test.js

```diff
@@ -337,7 +337,7 @@ module('Integration | Component | table', function(hooks) {
 
     let options = merge({}, Options, { showTotals: { subtotal: 'dateTime' } });
 
-    Model[0].response.rows = ROWS.slice(0, 4);
+    set(Model, 'firstObject.response.rows', ROWS.slice(0, 4));
```

Comment:
What was the issue here?

Reply:
I was getting an error when the tests ran saying that because that property is being observed, it should be set with `Ember.set`